### DISCUSSION TITLE
Android

### DISCUSSION THREAD

**Discussion by a-electronix** [collin80/SavvyCAN#594] [General]
Hi, perhaps a stupid question. But would it be possible to build SavvyCAN to be used on Android?
Br.Martin

**Comment by collin80** [comment]
Is it possible? Yes. Should you do it? Not really. I've built test binaries for Android in the past. It can work but the GUI is just not set up very well for a tablet or phone so things look really tiny and are hard to use. I'm thinking about making a webassembly / webgl application with a GUI meant to run on phones, tablets, and web browsers in general. Such an app would then be easier to use. But, nothing has been even started yet.

**Comment by a-electronix** [comment]
Thank you Collin for the quick reply,

Then I know, love the work with SavvyCAN.

Best Regards
_Martin_

2023-02-28 18:29 skrev Collin Kidder:

> Is it possible? Yes. Should you do it? Not really. I've built test 
> binaries for Android in the past. It can work but the GUI is just not 
> set up very well for a tablet or phone so things look really tiny and 
> are hard to use. I'm thinking about making a webassembly / webgl 
> application with a GUI meant to run on phones, tablets, and web 
> browsers in general. Such an app would then be easier to use. But, 
> nothing has been even started yet.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/collin80/SavvyCAN/discussions/594#discussioncomment-5155741
[2]
